### DISCUSSION TITLE
Typing quotes adds more quotes instead of completing existing quotes

### DISCUSSION THREAD

**Discussion by xxc-zsz** [microsoft/pylance-release#4544] [Enhancement]
If I type one quote VS Code will add a pair with the cursor in the middle. When I type a string between them and then type another quote, VS Code will register there is a quote there already and just move the cursor to the right of it. Perfect.

However, when I use triple quotes, VS will add more quotes between them if I type anything in the middle instead of just moving the cursor to the right of what’s already there.

why are they different? The latter should also act like one quote.

**Comment by xxc-zsz** [comment]
Related questions on [reddit](https://www.reddit.com/r/vscode/comments/13ztp3k/typing_quotes_adds_more_quotes_instead_of/)

**Comment by debonte** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
